docs/how-to-guides/themes/create-block-theme.md

```diff
@@ -1,33 +1,44 @@
 # Create a block theme
 
 The purpose of this tutorial is to show how to create a block theme and help theme developers transition to full site editing.
+It is recommended that you first read the [block theme overview](/docs/how-to-guides/themes/block-theme-overview.md).
 
 You will learn about the required files, how to combine templates and template parts, how to add presets for global styles, and how to add blocks and export the templates in the site editor.
 
 Full site editing is an experimental feature and the workflow in this tutorial is likely to change.
 
-This tutorial is up to date as of Gutenberg version 9.1.
+This tutorial is up to date as of Gutenberg version 10.6.
```

Comment:
Added 10.6 here tentatively because I wanted the tutorial update to include the changes to theme.json.